fetools-app/src/pages/ColorPicker.jsx

```diff
@@ -37,5 +47,19 @@ export default function ColorPicker() {
 
     )
 
+
+  function handleOnPageLoad(){
+    console.log(searchParams.get('color')!==null?true:false)
+    if(searchParams.get('color')!==null){
+      navigate(`/color-picker?color=${encodeURIComponent(searchParams.get('color'))}`)
```

Comment:
Why are you navigating here? I might be missing something, but I think this function can be removed entirely.